content/docs/user-guide/interactivity/user-interface/canvases/accessing-ui-canvas-runtime.md

```diff
@@ -0,0 +1,34 @@
+---
+linkTitle: Accessing UI Elements at Runtime
+description: ' Manipulate a UI Canvas at runtime. '
+title: Accessing UI Elements at Runtime
+weight: 900
+---
+
+Once you've loaded a UI Canvas, you can communicate with its UI elements at runtime by creating [Script Canvas](/docs/user-guide/scripting/script-canvas/get-started) graphs, [Lua scripts](/docs/user-guide/scripting/lua/ebus) or by using [EBuses](/docs/user-guide/programming/ebus) directly from C++.
+
+**Communicating with components attached to other UI elements**
+
+You can communicate with a UI element only after it has been activated. If you have a Script Canvas graph or a Lua script that is attached to a UI element that needs to get or set information on another UI element or on its owning UI canvas, your script should wait until all the UI elements are activated and until their parent and UI canvas references are fixed up. It is recommended to connect to the **UI Activation bus** and wait for the **In-game Post-activate event** before requesting the relevant information.
+
+{{< important >}}
+To ensure that UI elements are not accessed before they have been activated, it is recommended to connect to the **UI Activation bus** and subscribe to the **In-game Post-activate event** which only executes after all UI elements have been activated and their parent and UI canvas references fixed up.
+{{< /important >}}
+
+The following example shows how to use the UI Initialization bus in a Lua script that's attached to a UI element.
+
+```lua
+function samplescript:OnActivate()
+    -- Connect to the UI Initialization bus to receive events
+    self.initializationHandler = UiInitializationBus.Connect(self, self.entityId);
+end
+
+function samplescript:InGamePostActivate()
+    -- All UI elements have now been activated and their parent and UI canvas references fixed up
+    self.initializationHandler:Disconnect()
+    self.StartButtonHandler = UiButtonNotificationBus.Connect(self, self.Properties.StartButton)
+end
+```
+
+The following example shows how to use the UI Initialization bus in a Script Canvas graph that's attached to a UI element.
+![UI Activation Node](/images/user-guide/interactivity/user-interface/editor/ui-initialization-sc.png)
```

Comment:
```suggestion

    ![UI Activation Node](/images/user-guide/interactivity/user-interface/editor/ui-initialization-sc.png)
```
I believe this change is necessary to have the image indented properly.

Reply:
Indentation like that only works in certain scenarios, like when you're adding text or images to a list. Since there's no list here, I think it looks fine without any indentation. In fact, the more width the better for a wide image such as this one.

Reply:
Ok, fine to disregard this.